File: client/hrms-client/src/services/SchoolService.js
import axios from "axios"

const url = "http://localhost:4000/api/schools/"

const add = async (school)=>{
      const res = await axios.post(url+"add",school)

      return res.data
}

const updateSchool = async (id,schoolName,department,startYear,graduatedYear)=>{
      const res = await axios.post(url+"updateSchool?department="+department+"&graduatedYear="+graduatedYear+"&id="+id+"&schoolName="+schoolName+"&startYear="+startYear)

      return res.data
}

const deleteSchool = async (id)=>{
      const res = await axios.post(url+"deleteSchool?id="+id)

      return res.data
}


const service = {add,updateSchool,deleteSchool}

export default service

File: server/hrms/src/main/java/kodlamaio/hrms/business/abstracts/SchoolService.java
package kodlamaio.hrms.business.abstracts;

import kodlamaio.hrms.core.utilities.results.Result;
import kodlamaio.hrms.entities.concretes.School;

public interface SchoolService {
    Result add(School school);
    Result getByJobSeeker_id(int id);
    Result updateSchool(int id,String schoolName,String department,int startYear,String graduatedYear);
    Result deleteSchool(int id);
}
